src/main/index.js

```diff
@@ -5,9 +5,12 @@ import configureMenu, { dockMenu } from './menus'
 import createWindow, { windows } from './createWindow'
 // import { autoUpdater } from "electron-updater"
 
-const openFilesCache = []
+var openFilesCache = []
```

Comment:
Thanks for this PR.

`var` is not recommended, please use `let` to declare variables which will be changed latter.